server/routes/idea.js

```diff
@@ -1,44 +1,46 @@
+/* eslint-disable consistent-return */
```

Comment:
You might want to just create a .eslint.js file in the routes folder to disable this rule for all files in the folder.